platforms/anchor-platform/api-reference/callbacks/get-rates.api.mdx

```diff
@@ -152,9 +153,29 @@ either `sell_delivery_method` and `buy_delivery_method` may be included in reque
         <div>
           
           
+
           Success.
-          
-          
+        <Alert>
+
+            Upon receiving the response, the Anchor Platform will validate the amuont and price of the response.
+
+            If the validation fails, the Platform will respond to the client application's request with a HTTP status
+            code of `502 Bad Gateway`.
+
+        </Alert>
+
+        The `sell_mount`, `buy_amount`, `price`, and `fee` are validated as follows:
```

Comment:
I believe this file is generated from the `main-callbacks.yaml` file. Should we be editing that instead?

Reply:
Updated.